frontend/src/container/PipelinePage/tests/PipelinePageLayout.test.tsx

```diff
@@ -0,0 +1,40 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import i18n from 'ReactI18';
+import store from 'store';
+
+import PipelinePageLayout from '../Layouts';
+import { matchMedia } from './AddNewPipeline.test';
+
+beforeAll(() => {
+	matchMedia();
+});
+
+const queryClient = new QueryClient({
```

Comment:
Because, When the Test case is initialized at that moment we need to pass the query client. else Test case will fail as we are using API integration at same level. @palashgdev

Reply:
make this as util so you don't need to write same code everywhere